packages/api/models/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const sequelize = require('../helper/sequelize.js');
+const basename = path.basename(__filename);
+const db = {};
+
+// Instantiate all of the models defined in this directory
+// Then store them on the db object
+fs
+  .readdirSync(__dirname)
+  .filter(file => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
```

Comment:
This would be simpler with `path.extname` matcher as you already are requiring `path` at the top of the file.

https://nodejs.org/api/path.html#path_path_extname_path

Something like this also using implicit return syntax from es6 fat arrow function.

```javascript
.filter(file => file !== basename && path.extname(file) === '.js'
```